user-manual/import-export/upload-digital-object.rst

```diff
@@ -362,7 +434,8 @@ Edit reference and thumbnail representations
 6. If you wish to use a different image as the :term:`thumbnail` or
    :term:`reference <reference display copy>` version this is also performed
    from the Edit digital object screen. First click delete in Reference
-   representation or Thumbnail area.
+   representation or Thumbnail area. **Note: this functionality is only available
```

Comment:
I just did a quick test, and this does seem to be supported for authority records. I was able to upload a DO to an authority, delete the thumbnail, upload a different thumbnail, and then see my new thumbnail in the authority record browse page.